packages/web/src/services/audius-sdk/discoveryNodeSelector.ts

```diff
@@ -3,7 +3,70 @@ import { DiscoveryNodeSelectorService } from '@audius/common'
 import { env } from '../env'
 import { remoteConfigInstance } from '../remote-config/remote-config-instance'
 
+const DISCOVERY_PROVIDER_TIMESTAMP = '@audius/libs:discovery-node-timestamp'
+const CACHE_TTL = 24 * 60 * 1000 // 24 hours
+
+type CachedDiscoveryNodeTimestamp =
+  | {
+      endpoint?: string
+      timestamp?: number
+    }
+  | null
+  | undefined
+
+const getCachedDiscoveryNode = () => {
+  const cached = localStorage.getItem(DISCOVERY_PROVIDER_TIMESTAMP)
```

Comment:
nit: could use our `get/setExpiringJSONValue` methods from the LocalStorage class